src/Http/Controllers/AccountController.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace dnj\Account\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use dnj\Account\AccountManager;
+use dnj\Account\Contracts\AccountStatus;
+use dnj\Account\Http\Requests\AccountRequest;
+use dnj\Account\Http\Requests\CreateNewAccountRequest;
+use dnj\Account\Http\Resources\AccountResource;
+use dnj\Number\Number;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Http\Request;
+
+class AccountController extends Controller {
+	public AccountManager $account_manager;
+	
+	public function __construct ( AccountManager $account_manager ) {
```

Comment:
```suggestion
	public function __construct ( protected AccountManager $accountManager ) {
```

We use php 8.1, you can code less.
for naming variables use camelCase